### PR TITLE
Refactored navigation component

### LINKED ISSUE #1963: Refactor Navigation Storybook
**Is your feature request related to a problem? Please describe.**
In our Psammead storybook for navigation, we currently have an example of navigations and brands together, however, this can be misleading.

image

**Describe the solution you'd like**
Separate `igbo with Brand` into an independent container in storybook called `NavigationsWithBrands`.

**Describe alternatives you've considered**
If deemed not necessary, please close issue.

**Testing notes**
`igbo with Brand` should be in an independent container in storybook.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #1963 

**Overall change:** _
Refactored ```Navigation``` component by splitting it into ```with brand``` and ```without brand``` folders and moving/refactoring associated stories into these folders.

**Code changes:**
-  Removed ```brand``` parameter from ```navStoriesData.map```
- Added ```igbo with Brand``` to ```with brand``` folder and renamed to just ```igbo```
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-navigation/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 <!-- prettier-ignore -->
 | Version | Description |
+| 2.2.11 | [PR#1963](https://github.com/bbc/psammead/pull/2092) Refactored navigation component |
 |---------|-------------|
```
Comment (thekp) [inline packages/components/psammead-navigation/CHANGELOG.md:5]: move this one line up

File: packages/components/psammead-navigation/src/index.stories.jsx [hunk 2]
```diff
@@ -154,17 +154,34 @@ const navigationStory = (skipLinkText, currentPageText, navData, brand) =>
     ),
   });
 
-const stories = storiesOf('Components|Navigation', module)
+const stories = storiesOf('Components|Navigation/without brand', module)
```
Comment (thekp) [inline packages/components/psammead-navigation/src/index.stories.jsx]:
```suggestion
const storiesWithoutBrand = storiesOf('Components|Navigation/without brand', module)
```

Comment (hotinglok) [reply packages/components/psammead-navigation/src/index.stories.jsx]: Resolved in latest commit

**Review by oluoluoxenfree — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]
LGTM, shows nicely in local storybook too

**Comment by chris-hinds** [comment]
This doesn't need testing

**Review by dr3 — APPROVED** [review]

**Pull request merged**